packages/smart-contracts/artifacts/verification/ERC20SwapToPay.sol/ERC20SwapToPay.sol

```diff
@@ -66,7 +66,7 @@ contract ERC20SwapToPay is Ownable {
   function approvePaymentProxyToSpend(address _erc20Address) public {
```

Comment:
No need to review these files, they are mostly copies of the main code used for Etherscan verification.

Reply:
is it required to version the verification files? just asking

Reply:
@benjlevesque, it's useful to have them somewhere for quick access and re-use. Shall I put them somewhere else?

Reply:
no, it's fine I think